src/components/TodoContainer.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import TodosList from "./TodosList";
+import Header from "./Header";
+import InputTodo from "./InputTodo";
+import { v4 as uuidv4 } from "uuid";
+
+class TodoContainer extends React.Component {
+  state = {
+    todos: [
+      {
```

Comment:
- [ ] The new tasks are not saved on localStorage please add a function to do that.